timeseries-streaming/timeseries-java-applications/Adapters/src/main/java/com/google/dataflow/sample/timeseriesflow/adaptors/fsi/data/cme/ConvertCMETOBRowToTSDataPoints.java

```diff
@@ -0,0 +1,247 @@
+/*
```

Comment:
The name of this class is a bit of a mouthful @rezarokni @BhupiSindhwani could we discuss nomenclature a bit? I think maybe moving more of it up into the package name might help like `com.google.dataflow.sample.timeseriesflow.adaptors/cme/smartstream/tob/TSDataPoint.java` might be a bit more readable overall.